core/src/serde.rs

```diff
@@ -110,17 +71,12 @@ impl From<OracleConfig> for OracleConfigSerde {
                 c.ballot_parameters.contract_parameters,
             ),
             vote_parameters: c.ballot_parameters.vote_parameters,
-            ballot_token_owner_address: c.ballot_parameters.ballot_token_owner_address,
+            ballot_token_owner_address: AddressEncoder::new(network_prefix)
+                .address_to_str(&c.ballot_parameters.ballot_token_owner_address),
```

Comment:
It seems if `c.ballot_parameters.ballot_token_owner_address` will be `NetworkAddress` there will be no need for network prefix.